MakeDetailViewController.m

```diff
@@ -0,0 +1,86 @@
+//
+//  MakeDetailViewController.m
+//  Makes
+//
+//  Created by Mohamed Ragab on 1/10/17.
+//  Copyright © 2017 Mohamed Ragab. All rights reserved.
+//
+
+#import "MakeDetailViewController.h"
+#import "Masonry.h"
+
+
+@interface MakeDetailViewController ()
+
+@end
+
+@implementation MakeDetailViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    self.view.backgroundColor = [UIColor whiteColor];
+    
+    self.makeImageViewLarge = [[UIImageView alloc]initWithFrame:self.view.frame];
```

Comment:
Whenever you're deadline with a tableView or a collectionView .. as a good practice, you have to implement any subviews (ex: UITableViewCell) that are to be added in the viewController in separate classes .. where you handle their initialization, features' convenient attributes and constraints. Afterwards you instantiate it in the dedicated viewController and use it at your convenience. This ensures a more organized hierarchy to your code.
I recommend you read more about MVC design pattern, as a kickoff, here is a good reference: 
https://www.raywenderlich.com/46988/ios-design-patterns

(Stick to the MVC for now)